docs/councils/pythian/election/name-protocol.md

```diff
@@ -1,5 +1,8 @@
 - Wallet address (SPL/Solana)
 
+- For key custody, do you use a hardware wallet?
+  - Yes/No
```

Comment:
I think this question is too prescriptive. I think it should say something like: 

```
- Key Custody
  - Describe how you will custody your private key (hardware wallet, MPC provider such as fireblocks, etc). Please describe any additional opsec practices (e.g., the transaction policy for fireblocks) that you follow. 
```